clients/go-ethereum:local/geth.sh

```diff
@@ -66,6 +67,9 @@ if [ "$HIVE_FORK_SPURIOUS" != "" ]; then
 	chainconfig=`echo $chainconfig | jq ". + {\"eip158Block\": $HIVE_FORK_SPURIOUS}"`
 	chainconfig=`echo $chainconfig | jq ". + {\"eip155Block\": $HIVE_FORK_SPURIOUS}"`
 fi
+if [ "$HIVE_FORK_METROPOLIS" != ""]; then
+	chainconfig=`echo $chainconfig | jq ". + {\"MetropolisBlock\": $HIVE_FORK_METROPOLIS}"`
```

Comment:
This will probably be `metropolisBlock`, i.e. lowercase M. Perhaps it would be better to name it like that from the get go.